shared/common-adapters/list2.native.js

```diff
@@ -50,6 +50,7 @@ class List2<T> extends PureComponent<Props<T>, void> {
           onEndReached={this.props.onEndReached}
           windowSize={this.props.windowSize || 10}
           debug={false /* set to true to debug the list */}
+          contentContainerStyle={this.props.style}
```

Comment:
will this affect any other existing styles?

Reply:
No, nothing else uses this prop